Automattic/jetpack
# Shortcopy: update illustrations on plan headers

Fixes #11541

#### Changes proposed in this Pull Request:


* add new illustrations on My Plan header
* update styles for a proper layout

## Desktop 
<img width="1059" alt="Captura de pantalla 2019-03-21 a la(s) 18 29 16" src="https://user-images.githubusercontent.com/1041600/54786540-e57a7280-4c07-11e9-967b-aa6acb058bf6.png">

## Tablet
<img width="771" alt="Captura de pantalla 2019-03-21 a la(s) 18 29 32" src="https://user-images.githubusercontent.com/1041600/54786547-ea3f2680-4c07-11e9-9348-72c0e2453b52.png">

## Phone
<img width="525" alt="Captura de pantalla 2019-03-21 a la(s) 19 03 57" src="https://user-images.githubusercontent.com/1041600/54787970-3b511980-4c0c-11e9-856b-733755d2de29.png">

## Small Phone <480
<img width="478" alt="Captura de pantalla 2019-03-21 a la(s) 19 04 15" src="https://user-images.githubusercontent.com/1041600/54787979-43a95480-4c0c-11e9-9b18-137ff1ee861a.png">

#### Testing instructions:





* Go to the Jetpack dashboard and click My Plan
* Ensure that everything looks correct

#### Proposed changelog entry for your changes:


* Not needed. Part of Shortcopy project.
